tests/Stubs/LoggerStub.php

```diff
@@ -0,0 +1,81 @@
+<?php
+declare(strict_types=1);
+
+namespace Tests\LoyaltyCorp\Search\Stubs;
+
+use EoneoPay\Externals\Logger\Interfaces\LoggerInterface;
+use Throwable;
+
+final class LoggerStub implements LoggerInterface
+{
+
+    /**
+     * @inheritDoc
```

Comment:
{@inheritdoc} is what we use, there are different semantics for different inheritdoc styles and they all suck (with `{@inheritdoc}` being the least sucky